Dev

quent1utt opened the pull request.

quent1utt committed:
etape1

quent1utt committed:
Delete test

quent1utt committed:
Add files via upload

quent1utt committed:
Add files via upload

Création de l'arborescence et des différentes navigations présent au sein de l'application.
Installation de différents modules node (navigation, handle gesture, tabs...)
Création des composants Drawer et Header
Insertion des modules Playlist et Réseaux Sociaux correspondant au Gala. 
Ajout du composant Countdown
Test de mise en page pour les différentes views

quent1utt committed:
Add files via upload

arnaud9145 commented on app.json:
```diff
@@ -0,0 +1,30 @@
+{
+  "expo": {
+    "name": "tutonavigation",
```
Tu as oublié de renommer le projet ici

arnaud9145 commented on app.json:
```diff
@@ -0,0 +1,30 @@
+    "platforms": [
+      "ios",
+      "android",
+      "web"
```
Je crois pas que tu veuilles développer sur la plateforme web, si ?

arnaud9145 commented on navigation/DrawerNavigator.js:
```diff
@@ -0,0 +1,39 @@
+
+
+const DrawerNavigator = createDrawerNavigator({
+  Accueil: AccueilScreen, 
```
Si tu nommes bien tes variables, tu peux factoriser ton code ici.
En effet, en JavaScript le code suivant : 
{ variable: variable }
Peut être factoriser : 
{ variable }

Ainsi si Tu renommes l’importation de AccueilScreen en Accueil : 
import Accueil from '../Bundles/Home/Screen/AccueilScreen'

Tu peux simplifier ici en : 

const DrawerNavigator = createDrawerNavigator({
   Accueil,
    ...
}

Idem pour les autres

arnaud9145 commented on package.json:
```diff
@@ -0,0 +1,42 @@
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
```
Tu ne fais pas de web, cette ligne peut être enlevée

arnaud9145 commented on app.json:
```diff
@@ -0,0 +1,30 @@
+      "android",
+      "web"
+    ],
+    "version": "1.0.0",
```
Change la version de ton application, elle doit être supérieur à la version de l’application ionic gala utt

arnaud9145 commented on App.js:
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { createAppContainer, createSwitchNavigator } from 'react-navigation';
+import DrawerNavigator from './navigation/DrawerNavigator.js';
```
Le .js est optionnel, je te conseil de l’enlever pour simplifier l’écriture

arnaud9145 commented on App.js:
```diff
@@ -0,0 +1,15 @@
+}
+
+const AppSwitchNavigator = createSwitchNavigator({
+  Main: { screen: DrawerNavigator }
```
Je pense que tu peux simplifier cette expression

arnaud9145 commented on Bundles/Artists/NavigatorArtiste.js:
```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+import { Button, View, Text, TouchableOpacity } from 'react-native';
+import { createStackNavigator } from 'react-navigation-stack';
+import ArtistesScreen from './Screen/ArtistesScreen.js';
```
Je pense que c’est temporaire, mais tu devrais faire une page Artiste générale, qui prend en paramètres les paramètres de l’artiste, ce qui évite tout ce code dupliqué.

Surtout que plus tard tu récupéreras tous les artistes dynamiquement.

arnaud9145 reviewed (commented):
Quelques commentaires généraux : 

- Évite le code en français, habitue toi à coder en anglais. Par exemple ArtistesScreen -> ArtistsScreen
- Importer des fichier avec l’extensions .js alourdie ton code
- factorise tes styles. Essaye de définir dans un fichier tes styles que tu utilises souvent, puis appelle les chaque fois que tu en as besoin. Comme ça le jour où tu décides de changer le style de ton app, il suffit de changer un fichier et pas 50

Je te fais d’autres retours plus tard

quent1utt committed:
Mise à jour du dossier Artiste

Mise en page de la liste artiste (les artistes ne sont pas les officiels, ce sont des exemples ;-) )
Problème de transmissions de données à résoudre entre les Data Artistes et les pages détails artistes

quent1utt committed:
Delete Artiste1Screen.js

quent1utt committed:
Delete Artiste2Screen.js

quent1utt committed:
Add files via upload

quent1utt committed:
Mise à jour code

quent1utt committed:
Merge branch 'dev' of https://github.com/ungdev/cassiopee-mobile into dev
